examples/Azure_IoT_Central_Arduino_Portenta_H7/Azure_IoT_PnP_Template.cpp

```diff
@@ -0,0 +1,546 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <stdlib.h>
+#include <stdarg.h>
+
+#include <az_core.h>
+#include <az_iot.h>
+
+#include "AzureIoT.h"
+#include "Azure_IoT_PnP_Template.h"
+
+#include <az_precondition_internal.h>
+
+/* --- Defines --- */
+#define AZURE_PNP_MODEL_ID "dtmi:azureiot:devkit:freertos:Esp32AzureIotKit;1"
```

Comment:
double checking that we're sticking with this since it has similar sensors?

Reply:
Resolved, template is for testing purposes